client/lib/track-element-size/README.md

```diff
@@ -0,0 +1,90 @@
+Track Element Size
+==================
+
+This module is designed to provide utilities for tracking an element's size in the DOM.
+
+## Usage
+
+### `useWindowResizeRect`
+
+`useWindowResizeRect` is a React hook that tracks the size of an element's bounding client rect as
+the window is resized. It doesn't track modifications due to other reasons, such as layout changes.
+
+It returns both a ref, to be applied on the element to be measured, and a rect with the element's
+dimensions. This rect will be null at first, but will be updated with the initial values on a later
+render, as well as kept up to date after any window resizes.
+
+Here's a small sample:
+
+```JSX
+import { useWindowResizeRect } from 'lib/track-element-size';
+
+function SampleComponent() {
+  // Get ref, rect and subscribe to changes.
+  const [ ref, rect ] = useWindowResizeCallback( ref, resizeCallback );
```

Comment:
should this be `useWindowResizeRect`?

Reply:
Yes it should! And it shouldn't have any parameters. Good catch!

Reply:
Fixed in latest commit.